app/views/dashboard/apps/appId/team/teamView.tsx

```diff
@@ -48,123 +57,176 @@ const tiers = [
   const [isInviteNewUserOpen, setInviteNewUserOpen] = useState(false)
-  const [email, setEmail] = useState("")
+  const [inviteEmail, setInviteEmail] = useState("")
   const [radioSelectedValue, setRadioSelectedValue] = useState("ADMIN")
+  const [confirmationModalProps, setConfirmationModalProps] =
+    useState<ConfirmationModalPropsType>({
+      type: "options",
+      isActive: false,
+    })
+  const [confirmationModalTitle, setConfirmationModalTitle] = useState<string>("")
+  const [confirmationModalDescription, setConfirmationModalDescription] =
+    useState<string>("")
+  const [confirmationModalEmail, setConfirmationModalEmail] = useState<string>("")
 
   const transition = useTransition()
 
-  const mockedData = [
-    {
-      email: "testadmin@pokt.network",
-      roleName: "ADMIN",
-      accepted: true,
-    },
-    {
-      email: "testowner@pokt.network",
-      roleName: "OWNER",
-      accepted: true,
-    },
-    {
-      email: "test2@pokt.network",
-      roleName: "MEMBER",
-      accepted: true,
-    },
-  ]
+  const handleRemoveCancelClick = () => {
+    console.log("cancel")
+  }
+
+  const handleRemoveClick = () => {
+    console.log("remove")
+  }
+
+  const handleRemoveTryAgain = () => {
+    console.log("try again")
+  }
 
   const userRole = "ADMIN"
 
   return (
     <>
-      {state === "loading" && <Loader />}
-      {isInviteNewUserOpen && (
-        <Card>
-          <Title order={3}>Invite New User</Title>
-          <Form className="invite-new-user__form" method="post">
-            <TextInput
-              label="Email address"
-              name="email-address"
-              placeholder="new@server.com"
-              type="email"
-              value={email}
-              onChange={(e) => setEmail(e.target.value)}
-            />
-            <AppRadioCards
-              currentRadio={radioSelectedValue}
-              radioData={tiers}
-              setRadio={setRadioSelectedValue}
-            />
-          </Form>
-          <div className="invite-new-user__form__buttons-container">
-            <Button variant="outline" onClick={() => setInviteNewUserOpen(false)}>
-              Cancel
-            </Button>
-            <Button disabled={transition.state === "submitting" || email === ""}>
-              Send Invite
-            </Button>
-          </div>
-        </Card>
-      )}
-      <Table
-        paginate
-        columns={["Email", "Status", "Role", ""]}
-        data={mockedData.map(({ email, roleName, accepted }) => {
-          return {
-            id: email,
-            email: email,
-            status: {
-              element: <StatusTag accepted={accepted} />,
-              value: accepted ? "ACCEPTED" : "PENDING",
-            },
-            role: {
-              element: (
-                <div className="list__role">
-                  {userRole === "ADMIN" || userRole === "OWNER" ? (
+      <Form method="post">
```

Comment:
This view has multiple forms so wrapping the entire view inside a form is problematic